webapp/src/components/icons/raised_hand.tsx

```diff
@@ -13,14 +13,18 @@ export default function RaisedHandIcon(props: Props) {
     return (
         <svg
             {...props}
-            width='25px'
-            height='27px'
-            viewBox='0 0 25 27'
+            width='24px'
```

Comment:
(this is a comment for the whole thing) -- can't we use the compass-icons (and remove these svgs completely)? (I think I did that in another PR, but I've forgotten which)

Reply:
Maybe. To be honest I battled hard with certain compass icons when it came to precise sizing and alignment. They are adding layers of definitions that are making it extra difficult to customize. Having the svgs included keeps it simpler and as we have seen recently, doesn't depend on webapp versions, making the product self sufficient in a way. Overall I think this needs a broader discussion and some general review of the whole product as we are now in a rather inconsistent state.

Created https://mattermost.atlassian.net/browse/MM-48854